flutter/lib/src/flutter_enricher_event_processor.dart

```diff
@@ -115,19 +115,23 @@ class FlutterEnricherEventProcessor extends EventProcessor {
     );
   }
 
-  Map<String, dynamic> _getFlutterContext() {
+  Map<String, String> _getFlutterContext() {
     final currentLifecycle = _widgetsBinding?.lifecycleState;
-
-    return <String, dynamic>{
-      if (debugBrightnessOverride != null)
-        'debug_brightness_override': debugBrightnessOverride,
-      if (debugDefaultTargetPlatformOverride != null)
+    final debugPlatformOverride = debugDefaultTargetPlatformOverride;
+    final tempDebugBrightnessOverride = debugBrightnessOverride;
+    final initialLifecycleState = _window?.initialLifecycleState;
+    final defaultRouteName = _window?.defaultRouteName;
+
+    return <String, String>{
+      if (tempDebugBrightnessOverride != null)
+        'debug_brightness_override': describeEnum(tempDebugBrightnessOverride),
+      if (debugPlatformOverride != null)
         'debug_default_target_platform_override':
-            debugDefaultTargetPlatformOverride,
-      if (_window?.initialLifecycleState.isNotEmpty ?? false)
-        'initial_lifecycle_state': _window?.initialLifecycleState,
-      if (_window?.defaultRouteName.isNotEmpty ?? false)
-        'default_route_name': _window?.defaultRouteName,
+            describeEnum(debugPlatformOverride),
+      if (initialLifecycleState != null && initialLifecycleState.isNotEmpty)
+        'initial_lifecycle_state': initialLifecycleState,
+      if (defaultRouteName != null && defaultRouteName.isNotEmpty)
```

Comment:
whats about using the pattern `var?.isNotEmpty ?? false` as other places in the code

Reply:
This way there's Dart type inference and I don't need to add a `!`.